webapp/models.py

```diff
@@ -121,7 +129,7 @@ class JiraTask(db.Model, DateTimeMixin):
     jira_id: int = Column(Integer)
```

Comment:
since this should be a key, the type should be String

Reply:
Done 👍🏿